### PR TITLE
Add Neo 2 Layout data

### PR DESCRIPTION
**Pull request by kreibaum**
This is for https://github.com/gitpod-io/gitpod/issues/1363 where I have executed the generate-layout script on an Ubuntu 20.10. (Windows 10 does not support Neo 2 layouts afaik)

Since I just followed instructions and uploaded a generated file, I don't think the Eclipse Contributor Agreement (ECA) is required. If you think otherwise I can of course do that step.

#### What it does
Adds the result of

```yarn --cwd packages/core generate-layout --output src/common/keyboard/layouts/de-Neo_2-pc.json```

#### How to test

Choose Neo 2 Keyboard in (linux) keyboard settings. Then check if shortcuts work. (e.g. Copy + Paste) This is of course hard to do if you don't have the layout memorized :-/

#### Review checklist

- [ ] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

No. I have just uploaded the generated file.

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

**Comment by kreibaum** [comment]
Looks like the ECA check is included in your CI so I'll make sure to sign that.

**Comment by kreibaum** [comment]
I have now signed the ECA but I don't know how to retrigger the check.

**Comment by DukeNgn** [comment]
@kreibaum Hi, in order to pass the eca check, you will need to sign off your work in your git commit message also ([sign your work](https://github.com/eclipse-theia/theia/blob/master/CONTRIBUTING.md#sign-your-work))

File: packages/core/src/common/keyboard/layouts/de-Neo_2-pc.json [hunk 1]
```diff
@@ -0,0 +1 @@
...ue":"5","withShift":"«","withAltGr":"‹","withShiftAltGr":"♂"},"Digit6":{"value":"6","withShift":"$","withAltGr":"¢","withShiftAltGr":"⚥"},"Digit7":{"value":"7","withShift":"€","withAltGr":"¥","withShiftAltGr":"ϰ"},"Digit8":{"value":"8","withShift":"„","withAltGr":"‚","withShiftAltGr":"⟨"},"Digit9":{"value":"9","withShift":"“","withAltGr":"‘","withShiftAltGr":"⟩"},"Digit0":{"value":"0","withShift":"”","withAltGr":"’","withShiftAltGr":"₀"},"Minus":{"value":"-","withShift":"—","withShiftAltGr":"‑"},"Equal":{"value":"̀","withShift":"̧","withAltGr":"̊","withShiftAltGr":"̔"},"BracketLeft":{"value":"ß","withShift":"ẞ","withAltGr":"ſ","withShiftAltGr":"ς"},"BracketRight":{"value":"́","withShift":"̃","withAltGr":"̸","withShiftAltGr":"̓"},"Semicolon":{"value":"d","withShift":"D","withAltGr":":","withShiftAltGr":"δ"},"Quote":{"value":"y","withShift":"Y","withAltGr":"@","withShiftAltGr":"υ"},"Backquote":{"value":"̂","withShift":"̌","withAltGr":"↻","withShiftAltGr":"˞"},"Comma":{"value":",","withShift":"–","withAltGr":"\"","withShiftAltGr":"ϱ"},"Period":{"value":".","withShift":"•","withAltGr":"'","withShiftAltGr":"ϑ"},"Slash":{"value":"j","withShift":"J","withAltGr":";","withShiftAltGr":"θ"}}}
```
Comment (vince-fugnitto) [inline packages/core/src/common/keyboard/layouts/de-Neo_2-pc.json:1]: minor: please include a `newline` in addition to fixing the `sign-off`.

**Comment by vince-fugnitto** [comment]
@kreibaum do you still plan on updating the pull-request? 
Unfortunately, we cannot accept the changes without a proper `sign-off` in the commit message which consequently fails the `eca` (EclipseFoundation Contributor Agreement) check.

**Comment by kreibaum** [comment]
Yes I do. Sorry that I didn't get around to that yet. Will still take a while to find time.

If you prefer, you can close this PR and I'll create a new one later.

**Comment by vince-fugnitto** [comment]
> Yes I do. Sorry that I didn't get around to that yet. Will still take a while to find time.

There's no rush, we look forward to your contribution :)

**Comment by kreibaum** [comment]
With the Code IDE in Gitpod not having this problem and it becoming the default soon, I'll instead switch over to that and close this pull request.

**Pull request closed without merging**